packages/client/src/components/notebook/VariablePreview.tsx

```diff
@@ -116,12 +117,21 @@ export const VariablePreview = observer((props: VariablePreviewProps) => {
             }
         } else {
             const found = state.parseVariable(`{{${id}}}`);
-
-            return (
-                <Typography variant="body2" fontWeight="bold">
-                    {typeof found !== 'string' ? JSON.stringify(found) : found}{' '}
-                </Typography>
-            );
+            if (typeof found === 'string') {
```

Comment:
Add check to see if string is a json string

Reply:
Added in utility check here.